modules/persistent-storage-csi-cinder-storage-class.adoc

```diff
@@ -5,10 +5,15 @@
 [id="persistent-storage-csi-cinder_{context}"]
 = Making OpenStack Cinder CSI the default storage class
 
-In {product-title}, the default storage class references the in-tree Cinder driver. The storage class will default to referencing OpenStack Cinder CSI in a subsequent update of {product-title}. Volumes provisioned using the existing in-tree storage class are planned for migration to the OpenStack Cinder CSI storage class at that time.
-
 The OpenStack Cinder CSI driver uses the `cinder.csi.openstack.org` parameter key to support dynamic provisioning.
 
+In {product-title}, the default storage class references the in-tree Cinder driver.
+
+[NOTE]
+====
+With xref:../container_storage_interface/persistent-storage-csi-migration.adoc#persistent-storage-csi-migration[CSI automatic migration] enabled, volumes created using the default storage class actually use the CSI driver. However, these volumes are displayed like in-tree volumes, but have additional annotation stating that they have been migrated to CSI.
```

Comment:
- For feature gates, I've reached out to Michael Burke. He is the writer on this feature area to make these corrections.
- For the mention of the migration feature in in-tree topics, it's already there. Please see 3rd paragraph in the links provided in my 1st comment in this PR.
- I was told about the annotation to migrated volumes by Fabio. Is he wrong, or did I misunderstand him?

Reply:
@jsafrane I do see an annotation in volumes that have been migrated to the AWS EBS CSI: `pv.kubernetes.io/migrated-to: ebs.csi.aws.com`.

> If we did the CSI migration right, the volumes look exactly like in-tree volumes

I'm not sure if we should add this to the docs because some migrated PVs may look different after they have been migrated.

For instance, if the cluster has a long-lived PV with the beta region label `failure-domain.beta.kubernetes.io/region`, once this volume is migrated to CSI we will replace that beta label by the new one (`topology.kubernetes.io/region`). In addition do that, we'll also set the node affinity of that PV (if it's not set yet).

Reply:
Ok, there is an annotation, however, it's added to all PVs, not only those provisioned after the CSI migration is enabled. And the annotation is just a technical detail that we use to sync different Kubernetes components, we may choose a different annotation at any time.

About region labels, IMO even in-tree PVs will have `topology.kubernetes.io/region` label in 4.8, without any migration. It's related to Kubernets version, not migration.

Reply:
Thanks, @jsafrane  and @bertinatto for the details. It seems that I should drop the reference to the annotation, but maintain information about with migration enabled, the CSI driver is actually used. See my latest push.